tests/unittests/test_commands.py

```diff
@@ -105,5 +105,20 @@ def test_text_saves_params(self):
         self.assertEqual(output_text.fontsize, 64)
 
 
+class TestFontUtils(ShoebotTestCase):
+    @test_as_bot()
+    def test_fontnames_gives_output(self):
+        """
+        Verify that fontnames() gives a list as output.
+        """
+        output = fontnames()
+        # it's a list
+        self.assertTrue(isinstance(output, list))
+        # it's not empty
+        self.assertTrue(bool(output))
+        # contains a string
+        self.assertTrue(isinstance(output[0], str))
```

Comment:
Since, an empty list would mean things have gone badly wrong we skip to doing a regex test on the first element:

`assertRegexpMatches(output[0], r"(.*)\s(.*)")`

My process to work that out - looked @ the list of assertions here
https://kapeli.com/cheat_sheets/Python_unittest_Assertions.docset/Contents/Resources/Documents/index

Then prototyped something to check that regex matches against every font in my system:

```
import re

names =  [f"{family.get_name()} {face.get_face_name()}" for family in PangoCairo.font_map_get_default().list_families() for face in family.list_faces()]

# assertRegexpMatches(s, r)
for name in names:
    # after verifying this printed the list of fonts as expected, played with the regex below:
    if not (re.search(r"(.*)\s(.*)", name).groups()):
        print("!!", name)
```